src/unixfs/cat.rs

```diff
@@ -1,11 +1,11 @@
 use crate::{Error, Ipfs, IpfsTypes};
 use async_stream::stream;
 use cid::Cid;
+use core::borrow::Borrow;
+use core::fmt;
+use core::ops::Range;
 use futures::stream::Stream;
 use ipfs_unixfs::file::{visit::IdleFileVisit, FileReadFailed};
-use std::borrow::Borrow;
-use std::fmt;
-use std::ops::Range;
```

Comment:
Just noticed while handling local conflicts that these were probably extra. `ipfs` should not attempt to be no_std at this time.

Reply:
Indeed, the `ipfs` crate deals with input and output operations that are tied to the underlying OS